src/Admin/OptionsMenu.php

```diff
@@ -0,0 +1,483 @@
+		return sprintf( 'toplevel_page_%s', AMP_Options_Manager::OPTION_NAME );
+	}
+
+	/**
+	 * Enqueues settings page assets.
+	 *
+	 * @since 1.6.0
+	 *
+	 * @param string $hook_suffix The current admin page.
+	 */
+	public function enqueue_assets( $hook_suffix ) {
+		if ( $this->screen_handle() !== $hook_suffix ) {
+			return;
+		}
+
+		$asset_file   = AMP__DIR__ . '/assets/js/' . self::ASSET_HANDLE . '.asset.php';
+		$asset        = require $asset_file;
+		$dependencies = $asset['dependencies'];
+		$version      = $asset['version'];
+
+		wp_enqueue_script(
+			self::ASSET_HANDLE,
+			amp_get_asset_url( 'js/' . self::ASSET_HANDLE . '.js' ),
+			$dependencies,
+			$version,
+			true
+		);
+
+		wp_enqueue_style(
+			self::ASSET_HANDLE,
+			amp_get_asset_url( 'css/amp-settings.css' ),
+			[ $this->google_fonts->get_handle(), 'wp-components' ],
+			AMP__VERSION
+		);
+
+		wp_styles()->add_data( self::ASSET_HANDLE, 'rtl', 'replace' );
+
+		$theme           = wp_get_theme();
+		$is_reader_theme = in_array( get_stylesheet(), wp_list_pluck( ( new AMP_Reader_Themes() )->get_themes(), 'slug' ), true );
+
+		$js_data = [
+			'CURRENT_THEME'                      => [
+				'name'            => $theme->get( 'Name' ),
+				'description'     => $theme->get( 'Description' ),
+				'is_reader_theme' => $is_reader_theme,
+				'screenshot'      => $theme->get_screenshot(),
+				'url'             => $theme->get( 'ThemeURI' ),
+			],
+			'OPTIONS_REST_ENDPOINT'              => rest_url( 'amp/v1/options' ),
+			'READER_THEMES_REST_ENDPOINT'        => rest_url( 'amp/v1/reader-themes' ),
+			'THEME_IS_SUPPORTED'                 => in_array(
+				get_template(),
+				array_diff( AMP_Core_Theme_Sanitizer::get_supported_themes(), [ 'twentyten' ] ),
+				true
+			),
+			'THEME_SUPPORT_ARGS'                 => AMP_Theme_Support::get_theme_support_args(),
+			'THEME_SUPPORTS_READER_MODE'         => AMP_Theme_Support::supports_reader_mode(),
+			'UPDATES_NONCE'                      => wp_create_nonce( 'updates' ),
+			'USER_FIELD_DEVELOPER_TOOLS_ENABLED' => DevToolsUserAccess::USER_FIELD_DEVELOPER_TOOLS_ENABLED,
+			'USER_REST_ENDPOINT'                 => rest_url( 'wp/v2/users/me' ),
+		];
+
+		wp_add_inline_script(
+			self::ASSET_HANDLE,
+			sprintf(
+				'var ampSettings = %s;',
+				wp_json_encode( $js_data )
+			),
+			'before'
+		);
+
+		if ( function_exists( 'wp_set_script_translations' ) ) {
+			wp_set_script_translations( self::ASSET_HANDLE, 'amp' );
+		} elseif ( function_exists( 'wp_get_jed_locale_data' ) || function_exists( 'gutenberg_get_jed_locale_data' ) ) {
+			$locale_data  = function_exists( 'wp_get_jed_locale_data' ) ? wp_get_jed_locale_data( 'amp' ) : gutenberg_get_jed_locale_data( 'amp' );
+			$translations = wp_json_encode( $locale_data );
+
+			wp_add_inline_script(
+				self::ASSET_HANDLE,
+				'wp.i18n.setLocaleData( ' . $translations . ', "amp" );',
+				'after'
+			);
+		}
+	}
+
+	/**
+	 * Display Settings.
+	 *
+	 * @since 0.6
```

Comment:
This `@since` tag is off, since the code moved. Either remove or add a second entry.